djangocms_versioning/helpers.py

```diff
@@ -83,6 +82,7 @@ def register_versionadmin_proxy(versionable, admin_site=None):
                 versionable.version_model_proxy
             ),
             UserWarning,
+            stacklevel=2
```

Comment:
@fsbraun I don't know too much about this `stacklevel` attribute, but this is the only change that I can see here that's anywhere near the test failure.

And the test failing runs this function to try to register a versionable model with admin (I think that's the idea here at least). That test code is here;
https://github.com/django-cms/djangocms-versioning/blob/master/tests/test_admin.py#L334

Reply:
@marksweb I've also added ruff to #326. 

And right, I remember now: When I added `stacklevel=2` (I think CodeQL complained) the `test_admin.py` stated failing. I changed the test which checks if this warning is issued, including comparing the stack level parameter. 

See changed test here: https://github.com/django-cms/djangocms-versioning/blob/eaf5e9d4d1122d8ab6ce09fd20ef827993512667/tests/test_admin.py#L357-L360